examples/hubtype_demo/src/locales/en.ts

```diff
@@ -0,0 +1,2 @@
+export default {
+};
```

Comment:
can we define the copies in locales? in this way would be easier to maintain if we have to do modifications in this bot.

Reply:
Good  point, the idea is to have all the text inside Contentful. I create a new task for doing it

Reply:
@guinii @vanbasten17 remember that this is a demo. Let's not overengineer it.